src/LineageTree/lineageTree.py

```diff
@@ -2337,9 +1625,9 @@ def main_axes(self, time: int = None):
             list: A list that contains the array of eigenvalues and eigenvectors.
         """
         if time is None:
-            time = np.argmax(
-                [len(self.time_nodes[t]) for t in range(int(self.t_e))]
-            )
+            time = max(
+                [(len(nodes), time) for time, nodes in self.time_nodes.items()]
+            )[1]
```

Comment:
I was thinking about that and thought that there were a "better" way of doing it. There you go:
```python
time = max(self.time_nodes, key=lambda x: len(self.time_nodes[x]))
```